### PR TITLE
added profile page

### PR DESCRIPTION
**Pull request by stevenli39**
Screen Shot 2020-07-30 at 3 03 11 PM

### CODE REVIEW

File: src/assets/Line 2.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="282" height="1" viewBox="0 0 282 1" fill="none" xmlns="http://www.w3.org/2000/svg">
+<line y1="0.5" x2="282" y2="0.5" stroke="#AEC4F4"/>
+</svg>
```
Comment (dchen150) [inline src/assets/Line 2.svg:3]: what is this file for? if it's the line under the house, you can use a border instead

File: src/pages/member/MProfile.js [hunk 2]
```diff
@@ -0,0 +1,597 @@
...
+    const [favouriteEventName1, setFavouriteEventName1] = React.useState({})
+    const [favouriteEventName2, setFavouriteEventName2] = React.useState({}) 
+    const Fname = React.useState(props.user.fname); 
+    const Lname = React.useState(props.user.lname); 
+    const Year = React.useState(props.user.level); 
+    const Gender = React.useState(props.user.gender); 
+
+    const body = {
+        email: Email,
+        fname: Fname,
```
Comment (dchen150) [inline src/pages/member/MProfile.js:170]: i dont thikn you need a Fname constant declared at the top, you could just do fname: props.user.fname here

File: src/pages/member/MProfile.js [hunk 3]
```diff
@@ -0,0 +1,597 @@
...
+    const [Diet, setDiet] = React.useState(props.user.diet); 
+    const [eventsAttended, setEventsAttended] = React.useState();
+    const [recentEvent, setRecentEvent] = React.useState({});
+    const [favouriteEventIDs, setFavouriteEventIDs] = React.useState([]); 
+    const [favouriteEventName1, setFavouriteEventName1] = React.useState({})
+    const [favouriteEventName2, setFavouriteEventName2] = React.useState({}) 
+    const Fname = React.useState(props.user.fname); 
+    const Lname = React.useState(props.user.lname); 
+    const Year = React.useState(props.user.level); 
+    const Gender = React.useState(props.user.gender);
```
Comment (dchen150) [inline src/pages/member/MProfile.js:166]: no need to make these `useState` if their state is never being changed

File: src/pages/member/MProfile.js [hunk 4]
```diff
@@ -0,0 +1,597 @@
...
+                            if (recentEventCheckedInChecker) { 
+                                eventsAttendedCounter = eventsAttendedCounter + 1
+                                if (recentEventChecker === "") {
+                                    setRecentEvent(event)
+                                }
+                                recentEventChecker = "exists"
+                            } 
+                        }
+                    })
+                    console.log(eventsAttendedCounter)
```
Comment (dchen150) [inline src/pages/member/MProfile.js:245]: remove console.log

File: src/pages/member/MProfile.js (hunk 2 above)
Comment (stevenli39) [reply src/pages/member/MProfile.js:170]: I changed body fields to props.user.____ where I could, but for some reason I can't pass Year as props.user.level

**Review by dchen150 — APPROVED** [review]

**Review by dchen150 — APPROVED** [review]
lgtm

**Pull request merged**